testfixtures.go

```diff
@@ -335,7 +337,16 @@ func (l *Loader) Load() error {
 			if err := file.delete(tx, l.helper); err != nil {
 				return err
 			}
+		}
 
+		for _, file := range l.fixturesFiles {
+			modified, err := l.helper.isTableModified(tx, file.fileNameWithoutExtension())
```

Comment:
I wasn't sure how the isTableModified method works. So for now I repeated in both the delete and insert loops.

Reply:
`isTableModified` use table checksum functions (for the databases that support it) to check if the data changed after the initial load. This is done for performance optimizations.

We need to check it for both delete and insert actions, my only concern here that it'll come with a performance penalty (because we're running the same query twice).

I propose that we call this before both loops, maybe saving the result in a `map[string]bool` where the key is the table name, then we could just check that inside the loops.

Reply:
Ok that sounds good. Thanks for the explanation.

Reply:
I've updated this function to call isTableModified in a separate loop as you suggested.